book/src/external-provisioner.md

```diff
@@ -8,7 +8,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | Min K8s Version | Max K8s Version | Recommended K8s Version
 --|--|--|--|--|--|--|--
-[external-provisioner v1.3.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.3.0) | [release-1.3](https://github.com/kubernetes-csi/external-provisioner/tree/release-1.3) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-provisioner:v1.3.0 | v1.13 | - | v1.15
+[external-provisioner v1.3.1](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.3.1) | [release-1.3](https://github.com/kubernetes-csi/external-provisioner/tree/release-1.3) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-provisioner:v1.3.1 | v1.13 | - | v1.15
```

Comment:
Max K8s version for all versions < 1.4.0 should be v1.19

Reply:
that's why the table should be updated by the person who released the images :-P